go.mod

```diff
@@ -2,10 +2,40 @@ module github.com/bonitoo-io/influxdb-client-go-3
 
 go 1.20
 
-require github.com/stretchr/testify v1.8.2
+require (
+	github.com/deepmap/oapi-codegen v1.12.4
```

Comment:
Please remove code generator from source, we have to simplify our sources for better maintenance and PRs from community.

Reply:
Done! Currently are files copied into influx/model/oapi-codegen. Should I create vendor file for it instead ?